colibre-zooms/registration.py

```diff
@@ -746,11 +746,11 @@ def register_species_fractions(self, catalogue, aperture_sizes):
         # to be added for each.
 
         self.xgass_galaxy_selection = np.logical_and(
-            M_star > unyt.unyt_quantity(10 ** 9, "Solar_Mass"),
+            M_star > unyt.unyt_quantity(10**9, "Solar_Mass"),
```

Comment:
This looks a bit strange to me: isn't black supposed to use spaces in such exponent notation?

I.e. I'd expect the formatted version to be `10 ** 9`, not `10**9`